config.go

```diff
@@ -262,13 +272,13 @@ func ParseConfig(fn string) (Configuration, error) {
 		}
 	}
 
-    val, ok = input.Get("bridge", "rabbit_mq_auto_delete_queue")
-    if ok {
-        b, err := strconv.ParseBool(val)
-        if err == nil {
-            config.AMQPAutoDeleteQueue = b
-        }
-    }
+	val, ok = input.Get("bridge", "rabbit_mq_auto_delete_queue")
```

Comment:
please undo these changes, they are not part of the PR

Reply:
Yeah, that was just goformat. Lemme see what I can do about that.